index.html

```diff
@@ -207,34 +207,26 @@ <h3  class="about-boxes">Skills</h3>
     </ul>
 </section>
 
-
-
-
-
 <section class="foot-section">
     <div class="ornament-top"><img src="/pict-mobile/image_geometry_1.svg" alt=""></div>
 <h2 class="foot-section-h">Get started</h2>
 <p class="foot-section-p">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
 
 
-
-
-
     <form action="https://formspree.io/f/xaykaovn" method="post">
 		<div class="your-name-container" >
             <div>Your name</div>
-            <input type="text" name="" placeholder="Anastasiia" maxlength='30'  id="">
+            <input type="text" name="Your name" placeholder="Anastasiia" maxlength='30'  id="your-name" required>
         </div>
 
-
         <div class="email-container">
             <div>Your email</div>
-            <input type="email" placeholder="|"  name="your email" id=""  >
+            <input type="email" placeholder="|"  name="your email" id="email"  required >
```

Comment:
[OPTIONAL] AS it was missed by the previous reviewer, I am adding it as optional, but I highly recommend you take them into consideration, as it will improve the overall project:
- [ ] Please use a meaningful messages for your placeholder attribute, like you have done it for other input fields, currently it is empty, you can use `placeholder="Your email" `